scripts/src/osd/modules.lua

```diff
@@ -384,12 +384,16 @@ function qtdebuggerbuild()
 			if _OPTIONS["QT_HOME"]~=nil then
 				MOCTST = backtick(_OPTIONS["QT_HOME"] .. "/bin/moc --version 2>/dev/null")
 				if (MOCTST=='') then
-					MOCTST = backtick(_OPTIONS["QT_HOME"] .. "/libexec/moc --version 2>/dev/null")
+					local qt_host_libexecs = backtick(_OPTIONS["QT_HOME"] .. "/bin/qmake -query QT_HOST_LIBEXECS")
+					if (qt_host_libexecs=='') then
```

Comment:
May actually need to check this is a path as `qmake` can output messages to stdout, e.g. on non-existent property it outputs `**Unknown**`

Perhaps
```suggestion
					if (string.starts(qt_host_libexecs,"/")~=true) then
```